Casks/f/fl-studio.rb

```diff
@@ -1,10 +1,10 @@
 cask "fl-studio" do
-  version "24.1.2.4074"
-  sha256 "9171e4d1d11311744ab84e96a591d474a80d9fe83e3054682acfdafc21bba36a"
+  version "24.2.0.4161"
+  sha256 "8a417b52103f3a832f31b4ef3dbab808f878c03aa9cc025ffa6f553a24b7ab5a"
 
-  url "https://demodownload.image-line.com/flstudio/flstudio_mac_#{version}.dmg",
+  url "https://install.image-line.com/flstudio/flstudio_mac_#{version}.dmg",
```

Comment:
```suggestion
  url "https://support.image-line.com/redirect/flstudio_mac_installer",
```

If we unversion the download it seems to negotiate what it needs to pull from the right url after the redirect, but doesn't seem to work on the direct link.

Reply:
Thanks, I'll give it a try.

I handled the last few bumps, and it would always work fine eventually after a few days. I'm not sure what the exact issue is.

Reply:
@bevanjkay It seems that upstream uses the `_cfuvid` and `cf_clearance` [Cloudflare cookies](https://developers.cloudflare.com/fundamentals/reference/policies-compliances/cloudflare-cookies/#_cfuvid-for-rate-limiting-rules) to filter requests. Fetching the previous version 24.1.2.4074 no longer works on my end either. Could you confirm this by running `brew fetch`?

Unless we can find a workaround to provide a stable download, it might be better to disable the cask.

Reply:
Inconsistent results for me.

One system I can `brew fetch`, one I cannot, for version 24.1.2.4074

Reply:
I can `brew fetch`.

Reply:
Can't `brew fetch` the 24.1.2.4074 one, but it's working fine with the URL in the browser. Is this another Audacity situation?